tests/unit/iontests.js

```diff
@@ -51,15 +50,31 @@ define([
       'good/utf32.ion',
     ];
 
+    // For debugging, put single files in this list to have the test run only
+    // that file.   (But don't forget to clean this up on checkin!)
+    var debugList = [
```

Comment:
Thanks for the 'intern' pointer.  I tried using the grep option, but in this case it pollutes the output with hundreds of "SKIP:" lines.  I'm fine with removing this code if you like -- let me know...